dev/github/component_owners.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+# Mapping of Alluxio component to Github user(s) that will be automatically assigned for pull requests
+# Map<string, string[]>
+component-owner-mapping:
```

Comment:
add a default component as a fallback, which helps handle the case where someone adds a new directory but did not update this mapping?

Reply:
updated